### PR TITLE
Updating description for enableAutomaticUpdates and enableAutomaticOS…

### PR DESCRIPTION
**Pull request by Deepali-Gupta**
…Upgrade

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by Deepali-Gupta** [comment]
Hi,

I was just made aware that we have to make swagger changes on the Github repo. Can you please review this change? https://github.com/Azure/azure-rest-api-specs/pull/5503
I had earlier made a PR to the Core repo (https://msazure.visualstudio.com/DefaultCollection/One/_git/Compute-CPlat-Core/pullrequest/1521548).
Please let me know if anything else is required.

Thanks,
Deepali

**Comment by jhendrixMSFT** [comment]
@Deepali-Gupta this contains breaking changes in a stable API version, is that intended?

**Comment by Deepali-Gupta** [comment]
Hi, Do all tests pass after your update?

It looks like this is a breaking change, because you are changing some objects.

Do you really need this change with the same API version?

Thanks,

Hyonho

From: Deepali Gupta <Deepali.Gupta@microsoft.com>
Sent: Thursday, March 28, 2019 4:02 PM
To: Azure/azure-rest-api-specs <reply@reply.github.com>; Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>
Cc: Ravikiran Reddy <raredd@microsoft.com>; Hyonho Lee <Hyonho.Lee@microsoft.com>
Subject: RE: [Azure/azure-rest-api-specs] Updating description for enableAutomaticUpdates and enableAutomaticOS… (#5503)

Hi,

I was just made aware that we have to make swagger changes on the Github repo. Can you please review this change? https://github.com/Azure/azure-rest-api-specs/pull/5503<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F5503&data=02%7C01%7CHyonho.Lee%40microsoft.com%7C673380b6e2444a2dac9408d6b3d170c3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636894109477033601&sdata=GandvDeX%2FsXXrjH8tKbPeoywp8cAQ57HjQHk%2BfeEIq4%3D&reserved=0>
I had earlier made a PR to the Core repo (https://msazure.visualstudio.com/DefaultCollection/One/_git/Compute-CPlat-Core/pullrequest/1521548<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmsazure.visualstudio.com%2FDefaultCollection%2FOne%2F_git%2FCompute-CPlat-Core%2Fpullrequest%2F1521548&data=02%7C01%7CHyonho.Lee%40microsoft.com%7C673380b6e2444a2dac9408d6b3d170c3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636894109477043605&sdata=bKNV5QWyQBuPuse0RuVK%2F9Ii4Qnk0RjzG%2B67VCCmbPc%3D&reserved=0>).
Please let me know if anything else is required.

Thanks,
Deepali

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

File: specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json [hunk 1]
```diff
@@ -5577,7 +5577,35 @@
         },
         "enableAutomaticUpdates": {
           "type": "boolean",
-          "description": "Indicates whether virtual machine is enabled for automatic updates."
+          "description": "Indicates whether virtual machine is enabled for automatic windows updates. <br> <br> Default behavior is true."
```
Comment (KrisBash) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json]: Swagger definitions look fine for this. However, there are risks in adding new properties to an existing API version. When updated and non-updated clients have different models for the resource, updates can be ambiguous as to whether a missing property was an intentional removal or simply the result of a downlevel client. Do these need to be added to an existing API version or can they go out in a new version?

**Review by KrisBash — CHANGES_REQUESTED** [review]
Please see comment

**Review by KrisBash — APPROVED** [review]
Property additions are spurious and due to separation of definitions for two resource types to expose discrete definitions. LGTM

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
The 2019-03-01 compute.json swagger already exists in master.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
This is causing a merge conflict.  Please rebase on top of latest master.

**Comment by jhendrixMSFT** [comment]
Superseded by https://github.com/Azure/azure-rest-api-specs/pull/5697

**Pull request closed without merging**